.github/workflows/test_project.yml

```diff
@@ -0,0 +1,76 @@
+name: Add issues and PRs to project and move issues on label change
```

Comment:
File name `test_project` is inconsistent with those in https://github.com/pykale/pykale/tree/main/.github/workflows where dash is used rather than underscore. Moreover, the name is questionable. I think it should replace https://github.com/pykale/pykale/blob/main/.github/workflows/project.yml. Therefore, it maybe better to update that yml rather than creating a new one. I understand that you may want to test it but I think we can merge and test since this is not a stable released version.

Reply:
The formatting is also inconsistent with other workflow yml files.